woocommerce/woocommerce
# Privacy: Make scope of exporter and erasers clearer with more detailed friendly names

### All Submissions:


### Changes proposed in this Pull Request:

Closes #20218 .

### How to test the changes in this Pull Request:

1. Install and activate a friendly name plugin like https://github.com/allendav/wp-privacy-participating-plugins
2. Install this branch
3. Go to wp-admin > Tools > Export Personal Data and see the new names
4. Go to wp-admin > Tools > Erase Personal Data and see the new names

### Other information:


### Changelog entry

> Privacy: Make scope of exporter and erasers clearer with more detailed friendly names
